jung-algorithms/src/main/java/edu/uci/ics/jung/layout/algorithms/SpringBHVisitorLayoutAlgorithm.java

```diff
@@ -89,7 +105,7 @@ protected void addForceFrom(ForceObject other) {
                 }
               }
             };
-        tree.visit(nodeForceObject);
+        tree.acceptVisitor(nodeForceObject);
         Point f = nodeForceObject.f;
```

Comment:
better name for what really happens

Reply:
The nodeForceObject 'visits' the tree and its Nodes, so I think acceptVisitor is a correct name. I see your point from the perspective of what it does to the nodeForceObject as opposed to 'what the design pattern is', especially because there is only one purpose, as opposed to a general multipurpose 'visitor'.